packages/babel-traverse/test/scope.js

```diff
@@ -60,5 +73,13 @@ describe("scope", function () {
         _foo2: { }
       `).scope.generateUid("foo"), "_foo3");
     });
+
+    it("reference paths", function() {
+      const node = getIdentifierPath("function square(n) { return n * n}");
+      const referencePaths = node.context.scope.bindings.n.referencePaths;
```

Comment:
I actually wondering to myself what the difference betwee straight up path.scope vs path.context.scope then